src/libs/actions/PersonalDetails.js

```diff
@@ -37,7 +37,7 @@ function getDisplayName(login, personalDetail) {
     // If we have a number like +15857527441@expensify.sms then let's remove @expensify.sms and format it
```

Comment:
@s77rt I believe it would be best to get rid of this function, since it's actually not ever going to work - `allPersonalDetails` is now keyed by accountID, so we need to pass logins in here 😬 

I was thinking to fix in this PR, but since there's a lot of test steps I think a different PR may be better?

Reply:
Wanna just do minimum changes here to fix this bug and unblock the deploy for now?

Reply:
yes i think that's best 👍

Reply:
👍